chromium_src/chrome/browser/download/bubble/download_bubble_ui_controller.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "chrome/browser/download/bubble/download_bubble_ui_controller.h"
+
+#include "brave/browser/download/brave_download_commands.h"
+
+#define ProcessDownloadButtonPress ProcessDownloadButtonPressChromium
```

Comment:
nit: The common practice is to use `_ChromiumImpl`, I would side with consistency in this case.